docs/source/asr/speaker_diarization/intro.rst

```diff
@@ -0,0 +1,70 @@
+Speaker Diarization
+==================================
+
+Speaker Diarization (SD) is the task of segmenting audio recordings by speaker labels, that is Who Speaks When?
+
+.. image:: images/sd_pipeline.png
```

Comment:
The image looks great, but it has black text on black background (or maybe thats just my browser setting under dark mode). Just a heads up.

Reply:
It looks fine under dark model in my chrome though ugly...

Reply:
Yup darkmode is the issue, light mode seems fine.

Reply:
We are not supporting dark mode right in nemo docs? browser settings dark mode are hacks IMO